packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Tabs/tabs.js

```diff
@@ -0,0 +1,308 @@
+import React, {
+    Children,
+    useRef,
+    useEffect,
+    useCallback,
+    useState
+} from 'react';
+import {
+    Tabs as TabWrapper,
+    TabList,
+    Tab as TabHeader,
+    TabPanel
+} from 'react-tabs';
+import defaultClasses from './tabs.css';
+import { mergeClasses } from '../../../../../classify';
+import { arrayOf, number, oneOf, shape, string } from 'prop-types';
+
+/**
+ * Upper case the first letter of a string
+ *
+ * @param {string} string
+ * @returns {string}
+ */
+const upperCaseString = string => {
+    if (string) {
+        return string.charAt(0).toUpperCase() + string.slice(1);
+    }
+};
+
+/**
+ * Page Builder Tabs component.
+ *
+ * This component is part of the Page Builder / PWA integration. It can be consumed without Page Builder.
+ *
+ * @typedef Tabs
+ * @kind functional component
+ *
+ * @param {props} props React component props
+ *
+ * @returns {React.Element} A React component that displays a set of Tabs.
+ */
+const Tabs = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const navigationRef = useRef(null);
+    const [scrollElement, setScrollElement] = useState(null);
+    const [gradient, setGradient] = useState(null);
+    const isScrolling = useRef(false);
+    const clientX = useRef(0);
+    const scrollX = useRef(0);
+
+    const {
+        tabNavigationAlignment = 'left',
+        minHeight,
+        defaultIndex = 0,
+        headers = [],
+        textAlign,
+        border,
+        borderColor,
+        borderWidth,
+        borderRadius,
+        marginTop,
+        marginRight,
+        marginBottom,
+        marginLeft,
+        paddingTop,
+        paddingRight,
+        paddingBottom,
+        paddingLeft,
+        cssClasses = [],
+        children
+    } = props;
+
+    const onMouseDown = useCallback(event => {
+        isScrolling.current = true;
+        clientX.current = event.clientX;
+    }, []);
+
+    const onMouseUp = useCallback(() => {
+        isScrolling.current = false;
+    }, []);
+
+    const onMouseMove = useCallback(
```

Comment:
The convention for naming handlers is `handleEvent`, not `onEvent`. This way, handlers received from props (`onEvent`) can be distinguished from locally defined handlers (`handleEvent`). 

```suggestion
    const handleMouseMove = useCallback(
```

Recommend the same change for `onMouseDown` and `onMouseUp`, and don't forget to update other references in this file.

Reply:
Ah yeah, that makes more sense!
Resolved: https://github.com/magento/pwa-studio/pull/1872/commits/50cbc1f9d2749c2067d71b61e71393336208f5b1